app/src/main/java/com/example/calculator/CalculatorFragment.kt

```diff
@@ -240,11 +236,12 @@ class CalculatorFragment : Fragment(), OnClickListener {
             }
             val operatorIndex = isContainOperator(operatorList, inputData)
             if (operatorIndex != -1) {
-                Log.d(TAG, " Operator index : $operatorIndex")
                 val lastIndex = inputData.length
-                Log.d(TAG, "last index of input textview : $lastIndex")
                 val subString = inputData.substring((lastIndex - operatorIndex), lastIndex)
-                Log.d(TAG, "substring : $subString")
+                Log.d(
+                    TAG,
+                    "substring : $subString operator index: $operatorIndex Last index: $lastIndex"
+                )
                 if (subString.contains(".")) {
```

Comment:
make a constant String value for each constant like "." , "0." and other with their name and use that string here